### DISCUSSION TITLE
-----

### DISCUSSION THREAD

**Discussion by ethano8225** [aircrack-ng/aircrack-ng#2368] [General]
So basically, my school changed the password to the only good wifi and I'm not trying to sound whiny, but not having connection to the outside world whatsoever is a litttle ridiculous (cant text parents, cant text friends about whos giving who a ride). I have access to the password I want, it is on my chromebook "pre-set" but they disabled all of the means of getting off the chromebook directly, which left me thinking. Could I set up a fake network, name it identically to my school's wifi, have the connection bounce on first try so I have to send the password from my chromebook to my fake network, and then just capture it and boom I have it? maybe I have to decrypt something? I know it isnt going to be that simple but I've had this idea in me for years to get the password this way. A little bit of a dream of mine, especially because of how poorly they've treated me at my school, its like a littttle, dorky f u ;)
Thank you for taking the time to read this, even if you have no clue how to help :)

**Comment by aircrack-ng** [comment]
We do not help hacking others.
